cnb_image.go

```diff
@@ -18,36 +19,20 @@ import (
 // The working image could be any v1.Image,
 // but in practice will start off as a pointer to a locallayout.v1ImageFacade (or similar).
 type CNBImageCore struct {
-	v1.Image // the working image
-	Store    ImageStore
 	// required
-	repoName string
+	v1.Image // the working image
 	// optional
+	createdAt           time.Time
 	preferredMediaTypes MediaTypes
 	preserveHistory     bool
 	previousImage       v1.Image
 }
 
-type ImageStore interface {
-	Contains(identifier string) bool
-	Delete(identifier string) error
-	Save(image IdentifiableV1Image, withName string, withAdditionalNames ...string) (string, error)
-	SaveFile(image IdentifiableV1Image, withName string) (string, error)
-
-	DownloadLayersFor(identifier string) error
-	Layers() []v1.Layer
-}
```

Comment:
Originally I thought this could be an interface that all packages could implement, but since most packages would override the methods that call this interface it's not so useful